php/class-utils.php

```diff
@@ -417,6 +428,24 @@ public static function upgrade_3_0_1() {
 		update_option( Sync::META_KEYS['db_version'], get_plugin_instance()->version );
 	}
 
+	/**
+	 * Upgrade DB from v3.0.1 to v3.1.9.
+	 */
+	public static function upgrade_3_1_9() {
```

Comment:
It is interesting that the install path is a straightforward dbDelta, and the upgrade path goes through custom logic.
I would expect to go through exactly the same db scheme upgrade regardless if I uninstalled or not before upgrading

Reply:
Hum, not sure if I follow.

Reply:
sorry, after rereading the code this is a non-issue